internal/service/order.go

```diff
@@ -48,10 +75,88 @@ func (os *OrderService) Upload(ctx context.Context, order *models.Order) (*model
 		return nil, err
 	}
 
+	os.doAccrualForOrder(order.Number)
+
 	return order, nil
 }
 
 // ListUserOrders returns list of user orders
 func (os *OrderService) ListUserOrders(ctx context.Context, userID uint64) ([]models.Order, error) {
 	return os.repo.GetOrdersByUserID(ctx, userID)
 }
+
+// doAccrualForOrder performs accrual for order
+func (os *OrderService) doAccrualForOrder(order string) error {
```

Comment:
Стоит передавать конект как аргумент функции.
```suggestion
func (os *OrderService) doAccrualForOrder(ctx context.Context, order string) error {
```